mlibrary/catalog-browse
# [SEARCH-1605] Implement design for subject browse

# Overview
Next phase of Catalog Browse: Browse by subject! The view, fake data, and environment variable have been created to make everything happen.

This pull request closes [SEARCH-1605](https://tools.lib.umich.edu/jira/browse/SEARCH-1605).

## Testing
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Go to [Browse by subject](http://localhost:4567/subject)
  - Does information display?
  - If you edit `.lib/models/fake_subjects.yml`, does the information update?
